src/components/templates/ProductTemplate/ProductTemplate/ProductTemplate.tsx

```diff
@@ -1,37 +1,37 @@
 import React, { ReactNode } from 'react';
-import styled from 'styled-components';
 
 import { ProgressProps } from '../../../molecules/Progress/Progress';
 import { ProductTemplateTitle } from '../ProductTemplateTitle/ProductTemplateTitle';
 import { ProductTemplateHeader } from '../ProductTemplateHeader/ProductTemplateHeader';
+import FlexContainer from '../../../layout/FlexContainer/FlexContainer';
+import FlexRow from '../../../layout/FlexRow/FlexRow';
+import FlexCol from '../../../layout/FlexCol/FlexCol';
 
 export interface ProductTemplateProps {
   children: React.ReactNode;
   title: string;
   subtitle?: string;
   prevStep?: string | ReactNode;
   progress?: Pick<ProgressProps, 'currentStep' | 'totalSteps'>;
+  contentWidth?: number;
 }
 
-function ProductTemplate({ title, subtitle, children, prevStep, progress }: ProductTemplateProps) {
+function ProductTemplate({ title, subtitle, children, prevStep, progress, contentWidth = 6 }: ProductTemplateProps) {
   return (
-    <ProductTemplateContainer data-automation="ZA.ProductTemplate">
-      <ProductTemplateHeader prevStep={prevStep} progress={progress} />
-      <ProductTemplateTitle title={title} subtitle={subtitle} />
-      <ProductTemplateContent>{children}</ProductTemplateContent>
-    </ProductTemplateContainer>
+    <FlexContainer data-automation="ZA.ProductTemplate" className="mb-10">
+      <FlexRow>
+        <FlexCol>
+          <ProductTemplateHeader prevStep={prevStep} progress={progress} />
+          <ProductTemplateTitle title={title} subtitle={subtitle} />
+          <FlexRow justify="center">
+            <FlexCol m={10} xl={contentWidth}>
```

Comment:
I'm not sure about hardcoding the width here but I also didn't want to add the option of setting the width on every breakpoint due to that making us less consistent across products but I'm happy to change it.

Reply:
Sounds good to me. Let's try to keep it hardcoded as that's what Lianne suggested to have standarized